django/db/models/functions.py

```diff
@@ -182,13 +180,11 @@ def __init__(self, expression, pos, length=None, **extra):
             expressions.append(length)
         super(Substr, self).__init__(*expressions, **extra)
 
-    def as_sqlite(self, compiler, connection):
-        self.function = 'SUBSTR'
-        return super(Substr, self).as_sql(compiler, connection)
+    def as_sqlite(self, compiler, connection, **kwargs):
```

Comment:
I don't understand the reason for changes like this one. Why does the method need to accept unused `**kwargs`?